cmd/crank/beta/xpkg/xpkg.go

```diff
@@ -17,17 +17,23 @@ limitations under the License.
 // Package xpkg contains beta Crossplane packaging commands.
 package xpkg
 
-// Cmd contains commands for interacting with xpkgs.
+// Cmd contains commands for interacting with packages.
 type Cmd struct {
-	Init initCmd `cmd:"" help:"Initialize a package from a template."`
+	// Keep commands sorted alphabetically.
+	Init initCmd `cmd:"" help:"Initialize a new package from a template."`
 }
 
 // Help prints out the help for the xpkg command.
 func (c *Cmd) Help() string {
 	return `
-Crossplane can be extended with packages. Several types of packages exist,
-including providers and configurations. A package is an opinionated OCI image
-that contains everything needed to extend Crossplane. For more detailed
-information on packages, see https://docs.crossplane.io/latest/concepts/packages.
+Crossplane can be extended using packages. A Crossplane package is sometimes
+called an xpkg. Crossplane supports configuration, provider and function
+packages. 
```

Comment:
I think you forgot to update the wording here to match that of xpkg/xpkg.go:
```suggestion
Crossplane can be extended using packages. Crossplane packages are called xpkgs.
Crossplane supports configuration, provider and function packages. 
```

Reply:
I did, thanks!